src/plugins/textpattern/src/main/js/core/Settings.js

```diff
@@ -17,14 +17,25 @@ define(
       { start: '- ', cmd: 'InsertUnorderedList' }
     ];
 
+    var defaultForcedRootBlock = 'p';
+
     var getPatterns = function (editorSettings) {
       return editorSettings.textpattern_patterns !== undefined ?
         editorSettings.textpattern_patterns :
         defaultPatterns;
     };
 
+    var getForcedRootBlock = function (editorSettings) {
+      if (editorSettings.forced_root_block === 'div') {
```

Comment:
Why is this only using it if it is a `div`? Shouldn't it just use anything you set `forced_root_block` too?